Release for 0.11.0 (#367)
wpengine/atlas-content-modeler: CHANGELOG.md

# Atlas Content Modeler Changelog

## 0.11.0 - 2021-12-01
### Added

> - You can now set Featured Image when using Media Field.

Reviewer: Might be worth linking to https://www.wpgraphql.com/docs/media/ and/or the REST version?

Author: Updated.

> - You can now set Featured Image when using Media Field [WPGraphQL Media Doc](https://www.wpgraphql.com/docs/media/).

Reviewer 2: Tiny grammar tweak, updated link to `featuredImage` section of the docs:

```suggestion
- You can now set a Media Field as the [Featured Image](https://www.wpgraphql.com/docs/media/#query-a-post-with-its-featured-image).
```

- Ability to select choice in a list using the new Multiple Choice Field (beta).

### Fixed
- Post titles are now available in WPGraphQL responses.
- Prevent reserved taxonomy slugs from being used as taxonomy slug.
- Used consistent labels to describe the taxonomy ID.
- Changing model plural name now updates the sidebar menu item automatically.
- Issue where sidebar menu doesn't expand under certain conditions.

### Changed

> - Standardized WP_Error codes.

Reviewer: Can we call this out a bit more for anyone who might be using it?

Reviewer 2: They're internal REST APIs so unlikely to be in public use, but if we want to be more explicit perhaps something like:

```suggestion
- Standardized WP_Error codes for internal REST endpoints. All error statuses now use an acm_ prefix instead of a mix of wpe_ and atlas_content_modeler_.
```
